persistence/mongodb/src/test/scala/net/liftweb/mongodb/MongoDirectMongoClient.scala

```diff
@@ -50,15 +50,15 @@ class MongoDirectMongoClientSpec extends Specification with MongoTestKit {
 
       doc.put("name", "MongoSession")
       doc.put("type", "db")
-      doc.put("count", 1)
+      doc.put("count", 1: java.lang.Integer)
 
       doc2.put("name", "MongoSession")
       doc2.put("type", "db")
-      doc2.put("count", 1)
+      doc2.put("count", 1: java.lang.Integer)
 
       doc3.put("name", "MongoDB")
       doc3.put("type", "db")
-      doc3.put("count", 1)
+      doc3.put("count", 1: java.lang.Integer)
```

Comment:
I'm not quite sure, but the compiler now complains without it:

    type mismatch;
    [error]  found   : Int(1)
    [error]  required: Object
    [error] Note: an implicit exists from scala.Int => java.lang.Integer, but
    [error] methods inherited from Object are rendered ambiguous.  This is to avoid
    [error] a blanket implicit which would convert any scala.Int to any AnyRef.
    [error] You may wish to use a type ascription: `x: java.lang.Integer`.
    [error]       doc.put("count", 1)